scripts/test/run-tests.sh

```diff
@@ -31,7 +31,7 @@ UBUNTU_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Ubuntu-Focal
 DEBIAN_IMAGE_URL="https://downloads.mender.io/mender-convert/images/Debian-11-x86-64.img.gz"
 
 ## Auto-update
-UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/20.04/release/ubuntu-20.04.3-preinstalled-server-armhf+raspi.img.xz"
+UBUNTU_SERVER_RPI_IMAGE_URL="http://cdimage.ubuntu.com/ubuntu/releases/20.04/release/ubuntu-20.04.4-preinstalled-server-armhf+raspi.img.xz"
```

Comment:
This the bot should have handled :(

Have you looked at it?

Reply:
I think it was released today. The bot will not look at it until next week, or?

I did look, there was nothing from the bot today.

Reply:
Hmm... Let's keep this on our minds. I"ll have a quick look now